lib/metrics/context.js

```diff
@@ -247,9 +227,9 @@ module.exports = function (log, config) {
   }
 }
 
-function getKey (token, event) {
-  if (token && event) {
-    return [ token.uid.toString('hex'), token.id, event ].join(':')
+function getKey (token) {
+  if (token) {
+    return [ token.uid.toString('hex'), token.id ].join(':')
```

Comment:
> I also don't think this will be the last time that we have to deploy something
>  which causes this kind of disruption. So it may be worth getting in to the habit
> of documenting the disruptions somewhere official and annotating it on the graphs

👍  right on!